### PR TITLE
clarify Floating IPs are supported

### PR DESCRIPTION
**Pull request by ahrkrak**
## Description

clarify in this doc that floating IPs are supported

## Todos

- [ ] Tests
- [ ] Documentation
- [ ] Release note

## Release Note

```release-note
None required
```

### CODE REVIEW

**Comment by caseydavenport** [comment]
Might want a quick check from @neiljerram to confirm.

File: v3.1/getting-started/openstack/neutron-api.md [hunk 1]
```diff
@@ -134,12 +134,14 @@ All the attributes of security groups remain unchanged in {{site.prodname}}.
...
-floating IPs. Neither of these objects are supported by {{site.prodname}}: they
-simply aren't required. For more information, see [this document]({{site.baseurl}}/{{page.version}}/getting-started/openstack/connectivity).
+floating IPs. Routers are not supported by {{site.prodname}}: they
+simply aren't required. For more information, see [this document]({{site.baseurl}}/{{page.version}}/getting-started/openstack/connectivity). 
 
-Any attempt to create these objects will fail, as {{site.prodname}} does not set up
+Any attempt to create a Router object will fail, as {{site.prodname}} does not set up
 any Neutron L3 Agents.
 
+Floating IPs are supported at beta level. For more information, see [this document]({{site.baseurl}}/{{page.version}}/usage/openstack/floating-ips).
```
Comment (neiljerram) [inline v3.1/getting-started/openstack/neutron-api.md]: This para on floating IPs is good - thank you.

I think the previous two paras on routers should be replaced with:

```
Calico provides connectivity by default between all Neutron networks, regardless of whether there are Router objects between them in the Neutron data model.  See [this document]({{site.baseurl}}/{{page.version}}/usage/openstack/semantics) for a fuller explanation.  Where isolation of a particular Neutron network is desired, we recommend expressing that through security group rules.
```

and I suggest moving that after the para about floating IPs.

WDYT?

**Comment by emanic** [comment]
If we want this content to persist beyond v3.1, the change needs to be made in the master directory, as well @ahrkrak if you can

**Comment by neiljerram** [comment]
@caseydavenport Would you like to re-review this before I merge it?

**Pull request merged**